commcare-cloud/commcare_cloud/environment/schemas/postgresql.py

```diff
@@ -1,39 +1,102 @@
 import jsonobject
+import re
+
+import six
+
+from commcare_cloud.environment.constants import constants
+
+
+def alphanum_key(key):
+    """adapted from https://stackoverflow.com/a/2669120/240553"""
```

Comment:
Would be useful to add a line saying what this is for: "Used to generate a natural sorting keys"